Jenkinsfile

```diff
@@ -147,7 +140,8 @@ String unit_packages() {
                            'python36-tabulate '
         if (quickbuild()) {
             // TODO: these should be gotten from the Requires: of RPM
-            packages += " spdk-tools mercury boost-devel libisa-l_crypto"
+            packages += " spdk-tools mercury-2.0.0~rc1" +
+                        " boost-devel libisa-l_crypto libfabric-debuginfo"
```

Comment:
I guess this debuginfo did install because it's in our Nexus repo, alongside the main packages.  I don't think this would work for O/S `-debuginfo` packages though.